packages/terra-application-links/src/tabs/_TabMenu.jsx

```diff
@@ -107,7 +107,7 @@ class TabMenu extends React.Component {
         refCallback={this.setTargetRef}
         isHidden={this.props.isHidden}
         text={childText || this.props.intl.formatMessage({ id: 'Terra.application.tabs.more' })}
-        ariaLabel={this.props.intl.formatMessage({ id: 'Terra.application.tabs.moreButtonDescription' })}
+        a11yLabel={this.props.intl.formatMessage({ id: 'Terra.application.tabs.moreButtonDescription' })}
```

Comment:
Taking a closer look at [TabMenuDisplay](https://github.com/cerner/terra-framework/blob/1b0a930facae7c8e07db5711dfc968a734fbe181/packages/terra-application-links/src/tabs/_TabMenuDisplay.jsx), the ariaLabel prop is being used to [set the aria-label on the div](https://github.com/cerner/terra-framework/blob/1b0a930facae7c8e07db5711dfc968a734fbe181/packages/terra-application-links/src/tabs/_TabMenuDisplay.jsx#L135), not the icon.
This TabMenuDisplay prop should remain as `ariaLabel`.

Reply:
So is it better to just document the code with a comment noting that and merge so it doesn't come back again as something to fix?

Reply:
Assuming its usage is meaningful and not decorative, IconCaretDown should be updated ([at this line](https://github.com/cerner/terra-framework/blob/1b0a930facae7c8e07db5711dfc968a734fbe181/packages/terra-application-links/src/tabs/_TabMenuDisplay.jsx#L146)) to use the `a11yLabel` prop rather than TabMenuDisplay.